lua/treesitter-context.lua

```diff
@@ -577,6 +630,7 @@ api.nvim_command('command! TSContextDisable lua require("treesitter-context").di
 api.nvim_command('command! TSContextToggle  lua require("treesitter-context").toggleEnabled()')
 
 api.nvim_command('highlight default link TreesitterContext NormalFloat')
+api.nvim_command "highlight default link TreesitterContextLineNumber LineNr"
```

Comment:
One thing, could you use a style consistent with the rest? E.g. here change double quotes for single quotes and add parenthesis, and above in other diffs use single quotes. Thanks :)

Reply:
Done, sorry about that! Think my auto-formatter got run by accident on some lines.